### DISCUSSION TITLE
How to setup and get it up and running

### DISCUSSION THREAD

**Discussion by Danielix99** [nilsteampassnet/TeamPass#3940] [General]
Hi everyone,
since i've tried many times and in many ways to setup this project i would like to share how i got around the majority of issues.

The only good version to start from as a fresh install seems to be the 3.0.7 release
after setting this up from zero using debian12 php.8.2 and mysql8 you need to upgrade it to latest version
i followed the instruction on the website to directly update it from 3.0.7 to 3.0.9 without issues, since 3.0.9 is still bugged (tasks not working) i proceeded to upgrade it from 3.0.9 to the latest 3.0.10 commit, the upgrade process actually froze on databse update, after restarting the update again i got it working and got 3.0.10 version up and running.

I hope this will help anyone trying to setup it

**Comment by kikimora52** [comment]
Thank you for this info. I managed to get it working on 3.0.8 from a fresh install, however, now I am running into issues where the admin and managers cannot load the locally sync'd users from LDAP. I would like to upgrade, however, when I attempt an upgrade it says the admin user is not authorized. I have tried for an hour now. Would you please describe the steps you took to upgrade?

Rough steps I took and struggled with:
docker compose down
cd /tpDir
git pull <remote> <branch>
git checkout install
docker compose up -d
https://tpurl/install/upgrade.php
